src/utils/customFetch.ts

```diff
@@ -0,0 +1,15 @@
+import { HttpVerbs } from "src/@types/HttpVerbs";
+
+export function customFetch(
```

Comment:
What do you think of naming it `fetchJson`?

"custom" does not convey any meaning ^^

Reply:
Sounds good, actually, at first I had named it jsonFetch before naming it custom fetch, I'm still hesitating

Reply:
OK fixed!